src/main/java/com/faforever/client/fa/relay/ice/IceAdapterImpl.java

```diff
@@ -198,15 +199,12 @@ public void onGpgGameMessage(GpgGameMessageEvent event) {
         ProcessBuilder processBuilder = new ProcessBuilder();
         processBuilder.directory(workDirectory.toFile());
         processBuilder.command(cmd);
-        processBuilder.environment().put("LOG_DIR", preferencesService.getFafLogDirectory().resolve("iceAdapterLogs").toAbsolutePath().toString());
+        processBuilder.environment().put("LOG_DIR", preferencesService.getIceAdapterLogDirectory().toAbsolutePath().toString());
 
-        log.debug("Starting ICE adapter with command: {}", cmd);
+        log.info("Starting ICE adapter with command: {}", cmd);
         process = processBuilder.start();
-        Logger logger = LoggerFactory.getLogger("faf-ice-adapter");
-        OsUtils.gobbleLines(process.getInputStream(), s -> {
-        });
-        OsUtils.gobbleLines(process.getErrorStream(), s -> {
-        });
+        OsUtils.gobbleLines(process.getInputStream(), s -> advancedLogger.debug("  STDOUT <- " + s));
+        OsUtils.gobbleLines(process.getErrorStream(), s -> advancedLogger.debug("  STDERR <- " + s));
```

Comment:
Please remove the arrow, and maybe even the STD.

Reply:
Ok. I did this so the (few) lines coming from `IceAdapterImpl` are clearly distinguished from the STDOUT/ERR lines, but i guess having the two space indentation is enough?